app/routes.js

```diff
@@ -26,7 +29,7 @@ const addContext = ({ context, user, csrfToken }) => ({
 export const routes = (authProvider) => {
   const router = express.Router();
 
-  router.use('/health', healthRoutes);
+  healthRoutes({ router, dependencies: getHealthCheckDependencies(config), logger });
```

Comment:
do we need to pass in the config or can we just get it straight into the routerHelper?

Reply:
didn't work for some reason when I didn't pass it in..